vernac/g_vernac.mlg

```diff
@@ -923,6 +923,9 @@ GRAMMAR EXTEND Gram
   command:
     [ [ IDENT "Comments"; l = LIST0 comment -> { VernacSynPure (VernacComments l) }
 
+      | IDENT "Library"; IDENT "Attributes"; attr = quoted_attributes ->
```

Comment:
```suggestion
      | IDENT "Library"; IDENT "Attributes"; attr = attribute_list ->
```

I don't see the need for the ungainly `#[` syntax here.  It's not needed to make the command parse.

Reply:
If we remove the `#[ ... ]`, then we have a comma-separated list, why not (this is already used in `Constraints`, `Arguments` and `Combined Scheme`).

One thumb up, are there more?